### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Wiwaltill**
Update because of the new iconoir.css file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,21 @@ You can download any icon of the pack directly from https://iconoir.com or get t
...
+
+```html
+<link rel="stylesheet" href="https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.css">
+```
+
 Here is an example in HTML:
 
-```shell
+```html
 <span class="iconoir-hand-brake"></span>
```
Comment (peterhil) [inline README.md:47]: Why not use `<i>` element like other icon sets?

File: README.md [hunk 2]
```diff
@@ -33,11 +33,21 @@ You can download any icon of the pack directly from https://iconoir.com or get t
 ## Font
 
 Iconoir is available in .EOT, .SVG, .TTF, and .WOFF. Get them in the /fonts folder.
+If you want to use Iconoir in a HTML file, you just need to add the [iconoir.css](./fonts/iconoir.css) file.
+
+Import the CSS File:
+
+```html
+<link rel="stylesheet" href="https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.css">
```
Comment (peterhil) [inline README.md:41]: This overflows on smaller screens, use relative URL instead?

`<link rel="stylesheet" href="./iconoir/fonts/iconoir.css">`

File: README.md [hunk 3]
```diff
@@ -33,11 +33,21 @@ You can download any icon of the pack directly from https://iconoir.com or get t
 ## Font
 
 Iconoir is available in .EOT, .SVG, .TTF, and .WOFF. Get them in the /fonts folder.
```
Comment (peterhil) [inline README.md:35]: I would move this paragraph to the end of the section because the basic usage is just: 1) Link to the style sheet 2) Use on HTML markup.

**Maybe rephrase also:**

The webfonts are in the [/fonts](./fonts/) folder. Iconoir is available in EOT, SVG, TTF, and WOFF formats.

File: README.md (hunk 2 above)
Comment (peterhil) [reply README.md:41]: Or put into two lines:

```
<link rel="stylesheet"
      href="https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.css">
```

File: README.md (hunk 1 above)
Comment (Wiwaltill) [reply README.md:47]: Since so far only "span" was used instead of "i", I have kept it that way.

File: README.md (hunk 2 above)
Comment (Wiwaltill) [reply README.md:41]: The long version of the link is so that the CSS file does not have to be downloaded.

Comment (Wiwaltill) [reply README.md:40]: And on smaller devices you can scroll in the code line. Writing the code in 2 lines makes no difference and doesn't look nice.

File: README.md (hunk 3 above)
Comment (Wiwaltill) [reply README.md]: Does the updated version match your requests?
